doc/en/conf.py

```diff
@@ -433,29 +237,33 @@
     )
 ]
 
-# -- Options for towncrier_draft extension -----------------------------------
+# -- Options for towncrier_draft extension --------------------------------------------
+# https://sphinxcontrib-towncrier.readthedocs.io/en/latest/#how-to-use-this
 
 towncrier_draft_autoversion_mode = "draft"  # or: 'sphinx-version', 'sphinx-release'
 towncrier_draft_include_empty = True
 towncrier_draft_working_directory = PROJECT_ROOT_DIR
 towncrier_draft_config_path = "pyproject.toml"  # relative to cwd
 
-
 # -- Options for sphinx_issues extension -----------------------------------
+# https://github.com/sloria/sphinx-issues#installation-and-configuration
 
 issues_github_path = "pytest-dev/pytest"
 
+# -- Custom Read the Docs build configuration -----------------------------------------
+# (undocumented, relies on implementation details of Read the Docs' build)
```

Comment:
I think, it's documented, though. As in the env vars are.

Reply:
Indeed, added the relevant documentation links.